website/docs/discover.md

```diff
@@ -0,0 +1,318 @@
+# Discover Podman Desktop in 10 minutes
+
+Podman Desktop is the central place to work with containers as a developer.
+
+The main purpose of Podman Desktop is to use the Docker, Podman and Kubernetes APIs
+to communicate with containers engines and Kubernetes clusters to provide you a graphical
+and unified interface across all these resources.
+
+It also provides extensions to help you install and configure these resources:
+install the Podman engine locally, install Kubernetes clusters locally,
+configure access to remote Kubernetes clusters, configure access to images registries,
+install various CLIs, and more.
+
+## Architecture
+
+The containers technology is a Linux kernel's technology. On a Linux system, you will be able to run Docker and/or Podman
+directly. On other systems like Windows and macOS, you will need to run a Linux instance inside a virtual machine
+and run Docker or Podman on this Linux instance, then access them from your system.
+
+### Containers on Linux systems
+
+On Linux systems, Docker is running as a daemon and creates a socket so clients (including the docker cli) can access it through its API.
+The daemon receives commands from the clients, and executes them to build containers, store images, etc.
+
+Podman, on the other hand, is daemonless. This means that the clients (including the podman cli) access directly
+the system resources (containers, filesystem, etc), using a Podman library, to do these operations.
+
+Podman can also run on a Docker-compatibility mode.
+To run in this mode, a communication channel is created (a socket `/var/run/docker.sock` on macOS/Linux, a named pipe on Windows) and a Podman service runs, listening on this channel. This setup allows podman resources
+to be accessed in two ways. The first way is by connecting to the Podman channel (for CLIs which have been originally designed to work with Docker only)
+the exact same way the Docker socket is accessed. The second way is to use the podman cli (or the podman library), which accesses the resources directly.
+Podman and Docker can still cohabit on the same system, by exposing different communication channels.
+
+Podman desktop accesses the Docker daemon through the communication channel to work with Docker resources,
+and uses the podman cli to work with Podman resources.
+
+### Containers on non-Linux systems
+
+On Windows and macOS systems, a Linux system instance needs to be started inside a virtual machine and, on this Linux system, Podman
+needs to be started as a service, so it can be accessed from outside the virtual machine (you local system), through SSH.
+
+On these systems, Podman Desktop uses the _remote_ version of the podman cli, which communicates with the service inside the virtual machine through
```

Comment:
throughput the doc, should we make podman in code quotes such as: `podman` to indicate it's the cli we are talking about?